core/src/scene/sceneLoader.cpp

```diff
@@ -744,7 +744,12 @@ void SceneLoader::loadCameras(Node _cameras, Scene& _scene) {
             }
 
             if (Node vanishing = camera["vanishing_point"]) {
-                // TODO
+                if (vanishing.IsSequence() && vanishing.size() >= 2) {
```

Comment:
Right, I think it's fine as long as XY coords are available, we're lenient
for cases like this elsewhere.

On Mon, Apr 4, 2016 at 7:52 AM, Matt Blair <notifications@github.com> wrote:

> In core/src/scene/sceneLoader.cpp
> <https://github.com/tangrams/tangram-es/pull/657#discussion_r58388350>:
>
> > @@ -744,7 +744,12 @@ void SceneLoader::loadCameras(Node _cameras, Scene& _scene) {
> >              }
> >
> >              if (Node vanishing = camera["vanishing_point"]) {
> > -                // TODO
> > +                if (vanishing.IsSequence() && vanishing.size() >= 2) {
>
> Probably not super important, but I'm guessing that JS doesn't check the
> array length is == 2.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/tangrams/tangram-es/pull/657/files/da8b42facb6862621955d647a16716388beb9184#r58388350>
>